support/troubleshooting-operator-issues.adoc

```diff
@@ -0,0 +1,24 @@
+[id="troubleshooting-operator-issues"]
+= Troubleshooting Operator issues
+include::modules/common-attributes.adoc[]
+:context: troubleshooting-operator-issues
+
+toc::[]
+
+Operators are a method of packaging, deploying, and managing a Kubernetes application. They act like an extension of the software vendor’s engineering team, watching over a Kubernetes environment (such as {product-title}) and using its current state to make decisions in real time. Advanced Operators are designed to handle upgrades seamlessly, react to failures automatically, and not take shortcuts, like skipping a software backup process to save time.
```

Comment:
suggest just saying OCP here, since it is OCP-specific docs:

```
over an {product-title} environment and using
```

Reply:
What makes an Operator "Advanced"? Or can that word be removed?

Reply:
Great suggestions, thanks. Resolved.